preliminaries.tex

```diff
@@ -0,0 +1,132 @@
+
+\begin{theorem}
+  A level-$k$ summit requires a quorum size of at least
+
+  \begin{equation}
+    q = \frac{\boldsymbol{w}(V)}{2} + \frac{t_f}{2(1-2^{-k})}
+  \end{equation}
+
+  in order to remain valid for a fault tolerance threshold of $t_f$.
+\end{theorem}
+
+We always consider level-1 summits for OTF, so the target quorum size becomes $q_{OTF}=\boldsymbol{w}(V)/2+t_f$. We also target a low $t_f$, e.g.~1\% of total weight. For the given fault tolerance threshold, we obtain $q_{OTF} = 0.51 \times \boldsymbol{w}(V)$.
+
+A tick $i$ is said to be \emph{feasible} if $\boldsymbol{w}(V_i)\geq q_{OTF}$; otherwise, it is said to be \emph{infeasible}. Rounds that start on feasible ticks are also said to be feasible. That is because the rounds that do not satisfy this condition cannot be finalized on time. Only feasible rounds are eligible for rewards, and participating in an infeasible round does not earn any reward.
+
+Finally, we can define on-time finality:
+
+\begin{definition}[On-time finality]
+  A \PROP sent on $i$ is said to be \emph{finalized on time in $v$'s perspective} only if $q_{i,v} \geq q_{OTF}$ once $v$'s round ends.
+\end{definition}
+
+We introduce the function
+
+\begin{equation}
+  f_i(v) =
+  \begin{cases}
+    1 & \text{if } q_{i,v} \geq q_{OTF} \\
+    0 & \text{otherwise}
+  \end{cases}
+\end{equation}
+
+to be used later on in seigniorage.
+
+\subsubsection*{Eventual Finality}
+\label{sec:eventual-finality}
+
+This is the type of finality that is achieved in the long run, which is not restricted to in-round messages. Summits on a message can increase, because voting can also happen indirectly, through messages in the subsequent rounds. A message $\mu$ \emph{votes} for a \PROP $P$ if either of the following conditions are satisfied:
+
+\begin{enumerate}
+\def\labelenumi{\arabic{enumi}.}
+\item $\mu$ is $P$ itself.
+\item $\mu$ is a \PROP which is a main-descendant of $P$.
+\item $\mu$ is a \CONF or \WIT, voting explicitly for $P$.
+\item $\mu$ is a \CONF or \WIT, voting explicitly for a main-descendant of $P$.
+\end{enumerate}
+
+Therefore, a \PROP can be finalized at a later round, even though it is not finalized on time.
+
+\subsection{Incentives at a Higher Level}
+
+Given all the moving bits of Highway and blockchains in general, it is essential to paint in broad strokes the type of behavior we want to incentivize on the CasperLabs blockchain. We believe that having this the bigger picture in mind will make reasoning easier in the following steps of our design.
+
+On the CasperLabs blockchain, we want to incentivize validators to:
+
+\begin{itemize}
+\item Converge on the same round exponent:
+\item Operate with the minimum round exponent:
+\item Finalize a round with as much weight as possible before it ends:
+\item Fill in blocks with as much transactions as possible:
```

Comment:
Should these lines all end in full stops instead of semicolons?
Also:
```suggestion
\item Fill in blocks with as many transactions as possible.
```

Reply:
I was intending to add more sentences there, forgot to mention it's a draft, sorry